scripts/build_bench_precompiles.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Root directory
+SUBNET_EVM_PATH=$(
+    cd "$(dirname "${BASH_SOURCE[0]}")"
+    cd .. && pwd
+)
+
+# Load the versions
+source "$SUBNET_EVM_PATH"/scripts/versions.sh
+
+# Load the constants
+source "$SUBNET_EVM_PATH"/scripts/constants.sh
+
+go test ./precompile/contracts/... -bench=./... -covermode=atomic -timeout="10m" $@
```

Comment:
does enabling coverage interfere with benchmarking?

Reply:
I'll remove it since it will add overhead and doesn't seem necessary